src/commands/UCSanDiego.ts

```diff
@@ -0,0 +1,24 @@
+
+import { Message } from 'discord.js';
+import Command from '../Command';
+import { BotClient } from '../types';
+
+/**
+ * Command to correct the spelling of UC San Diego
+ */
+export default class UCSanDiegoCommand extends Command {
+  constructor(client: BotClient) {
+    super(client, {
+      name: 'ucsd',
+      enabled: true,
+      description: 'Corrects the spelling of our glorious university, UC San Diego.',
+      category: 'Uncategorized',
```

Comment:
You could have this category:

```suggestion
      category: 'Information',
```

Or this one:

```suggestion
      category: 'Meme',
```